app/_data/docs_nav_konnect.yml

```diff
@@ -75,6 +73,8 @@
               url: /runtime-manager/configuration/vaults
             - text: Set Up and Use a Vault in Konnect
               url: /runtime-manager/configuration/vaults/how-to
+    - text: Manage Runtime Groups using decK
+      url: /runtime-manager/declarative-config
```

Comment:
Moved this down because I don't know that we want "Manage Runtime Groups using decK" appearing above config & instances.